.dockerignore

```diff
@@ -16,9 +16,6 @@
 /.dockerignore
 /Dockerfile*
 
-# Environment
-/.env*
```

Comment:
:thought_balloon: Removing this line will cause `env.test` and `env.development` to show up in the Docker image. I don't think we want that or am I missing something?

Reply:
How does the .env file get into a docker build? Running the system failed due to it not being there. But maybe I'm missing something?